pkg/envinfo/envinfo.go

```diff
@@ -161,6 +166,9 @@ func (esi *EnvSpecificInfo) SetConfiguration(parameter string, value interface{}
 			} else {
 				esi.componentSettings.URL = &[]EnvInfoURL{urlValue}
 			}
+		case "push":
```

Comment:
please add a test case for this in `TestSetEnvInfo`

Reply:
done